streamflow/workflow/token.py

```diff
@@ -161,8 +162,12 @@ async def is_available(self, context: StreamFlowContext):
 class TerminationToken(Token):
     __slots__ = ()
 
-    def __init__(self):
-        super().__init__(None)
+    def __init__(self, value: Any = Status.COMPLETED):
+        if not isinstance(value, Status):
+            raise WorkflowExecutionException(
+                f"Termination token received an invalid value type {type(value)}. Accepted only Status."
```

Comment:
Termination token received an invalid value type {type(value)}: it should be of type `Status`.